src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -189,17 +189,32 @@
 
                     requestStack.push(stackElement);
 
-                    this.addEventListener("readystatechange", function() {
-                        if (self.readyState == 4) {
-                            stackElement.duration = new Date() - stackElement.start;
-                            stackElement.loading = false;
-                            stackElement.error = self.status < 200 || self.status >= 400;
-                            stackElement.profile = self.getResponseHeader("X-Debug-Token");
-                            stackElement.profilerUrl = self.getResponseHeader("X-Debug-Token-Link");
-
-                            Sfjs.renderAjaxRequests();
-                        }
-                    }, false);
+                    /* check for older browsers */
+                    if (!window.addEventListener) {
+                        this.attachEvent("readystatechange", function () {
```

Comment:
It can be nice to add a comment like that to help people to handerstand this part of code.
``` js
// Fix ie8&9, does not have XMLHttpRequest.addEventListener method.
// @see https://developer.mozilla.org/fr/docs/Web/API/Element.addEventListener#Internet_Explorer
```